xbmc/windowing/windows/WinSystemWin32DX.cpp

```diff
@@ -380,3 +381,41 @@ HRESULT APIENTRY HookOpenAdapter10_2(D3D10DDIARG_OPENADAPTER *pOpenData)
   }
   return hr;
 }
+
+bool CWinSystemWin32DX::IsHDRDisplay()
+{
+  if (CWIN32Util::GetWindowsHDRStatus() != HDR_STATUS::HDR_UNSUPPORTED)
+    return true;
+
+  return false;
```

Comment:
```suggestion
  return (GetOSHDRStatus() != HDR_STATUS::HDR_UNSUPPORTED);
```

Reply:
I like the simplification, however I keep `GetWindowsHDRStatus` because it's more direct: `GetOSHDRStatus` calls `GetWindowsHDRStatus` on Windows.

I'm going to change it.